packages/layouts-css/src/page.scss

```diff
@@ -83,21 +83,21 @@
     &-title-area {
       display: flex;
       flex-direction: column;
-      margin-bottom: $iui-baseline * 3;
+      margin-bottom: calc(var(--iui-size-s) * 3);
     }
 
     &-tools {
       display: flex;
       flex-direction: row;
-      margin-bottom: $iui-baseline * 2;
+      margin-bottom: calc(var(--iui-size-s) * 2);
       justify-content: space-between;
 
       &-left,
       &-right {
         display: flex;
         justify-content: flex-start;
         flex-direction: row;
-        gap: $iui-s;
+        gap: var(--iui-size-xs);
```

Comment:
Only now I understand that our example doesn't have several buttons or inputs in tools area to see how gap looks like. But it is not for this PR.